### PR TITLE
Feature language date choice

### PR DESCRIPTION
**Pull request by dkhasham**
related Ticket: https://synjira.atlassian.net/browse/CELDEV-235

### CODE REVIEW

**Comment by fpichler** [comment]
@msladek can you please review?

File: component/src/main/java/com/celements/calendar/Event.java [hunk 1]
```diff
@@ -167,7 +167,18 @@ public String getLocation() {
...
   @Override
   public String getDateString(String dateField, String format) {
-    SimpleDateFormat sdf = new SimpleDateFormat(format, new Locale(getLanguage()));
+    return getDateString(dateField, format, null);
+  }
+  
+  @Override
+  public String getDateString(String dateField, String format, String language) {
+    Locale localLanguage = null;
+    if((language == null) || (language == "")) {
```
Comment (msladek) [inline component/src/main/java/com/celements/calendar/Event.java]: use org.apache.commons.lang3.StringUtils.isBlank(str)

also, space missing after _if_, consider using save actions: http://intern.synventis.com/SynWiki/Eclipse-Java-Code-Style

**Comment by fpichler** [comment]
sorry for claiming code style again... some month ago we decided to favor google.guava instead of apache.lang3 where ever possible. Thus, please use com.google.common.base.Strings.isNullOrEmpty instead of org.apache.commons.lang3.StringUtils.isBlank
(to get the semantically equal use nullToEmpty(theString).trim().isEmtpy() which imho is not needed here for 'language')

**Comment by dkhasham** [comment]
changed to isNullOrEmpty check

**Pull request merged**